index.html 

```diff
@@ -0,0 +1,42 @@
+<! doctype html>
+<head>
+	<title> Bubble Tea site</title>
+</head>
+<body>
+	< div class=“topnav”>
+		<img src =“ ” ALT=“logo” Width=600 Height = 400”/>
```

Comment:
ALT needs to be alt
width and height need to be in style="width:600px; height:400px" format